src/test/java/com/commercetools/sync/categories/utils/categoryupdateactionutils/BuildAssetsUpdateActionsTest.java

```diff
@@ -192,14 +272,14 @@ void buildAssetsUpdateActions_WithOneExtraAsset_ShouldBuildAddAssetAction() {
             readObjectFromResource(CATEGORY_DRAFT_WITH_ASSETS_ABCD, CategoryDraft.class);
 
         final List<UpdateAction<Category>> updateActions =
-            buildAssetsUpdateActions(oldCategory, newCategoryDraft, SYNC_OPTIONS);
+            buildAssetsUpdateActions(oldCategory, newCategoryDraft, syncOptions);
 
         assertThat(updateActions).containsExactly(
             AddAsset.of(
                 AssetDraftBuilder.of(singletonList(AssetSourceBuilder.ofUri("uri").build()), ofEnglish("asset name"))
-                                 .key("d")
-                                 .tags(emptySet())
-                                 .build(), 3));
+                    .key("d")
```

Comment:
please try not to change code just for formatting, this is overriding the git history.

Reply:
I only fixed it according to checkstyle